src/main/kotlin/app/revanced/patches/youtube/layout/dimming/ShortsDimmingPatch.kt

```diff
@@ -9,7 +9,7 @@ import app.revanced.util.patch.BaseResourcePatch
 @Suppress("unused")
 object ShortsDimmingPatch : BaseResourcePatch(
     name = "Hide shorts dimming",
-    description = "Hide the dimming effect on the top and bottom of Shorts video at compile time.",
+    description = "Removes the dimming effect at the top and bottom of Shorts videos at compile time.",
```

Comment:
Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the dimming effect at the top and bottom of Shorts videos.",
```